python/src/equistore/labels.py

```diff
@@ -251,6 +251,48 @@ def asarray(self):
     def __contains__(self, label):
         return self.position(label) is not None
 
+    @staticmethod
+    def arange(name, *args):
```

Comment:
one comment, nice function. For now the function just create a label with one name and only one arange.
Maybe it would be also helpful to have the possibility to create a label with more names, something like 

```python
arange(names=['a1', 'a2'],start=[0,3],stop=[2,7],step=[1,2])-> Labels(['a1', 'a2'],np.array([[0,3], [1,5]])```

the function should handle the errors if the the two arange are not compatible